test/integ_tests/test_all_notebooks.py

```diff
@@ -24,13 +24,8 @@
 
 if os.environ.get("AWS_DEFAULT_REGION") == "eu-north-1" or os.environ.get("AWS_REGION") == "eu-north-1":
     EXTRA_EXCLUDES = [
-        "Quantum_machine_learning_in_Amazon_Braket_Hybrid_Jobs.ipynb",
-        "Using_PennyLane_with_Braket_Hybrid_Jobs.ipynb",
-        "Running_notebooks_as_hybrid_jobs.ipynb",
-        "2_Graph_optimization_with_QAOA.ipynb",
         "Using_The_Adjoint_Gradient_Result_Type.ipynb",
         "0_Getting_Started.ipynb",
```

Comment:
Can you also verify if Getting_Started notebook needs to be excluded?

Reply:
Yes, there is an issue around qiskit get_backends operates with the mocks.